examples/react-redux/public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" data-framework="react-redux">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="description" content="A TodoMVC written in React-Redux." />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <title>TodoMVC: React-Redux</title>
+    </head>
+    <body>
+        <section class="todoapp" id="root"></section>
+        <footer class="info">
+            <p>Click on input field to write your todo.</p>
+            <p>At least two characters are needed to be a valid entry.</p>
```

Comment:
These instructions are not added to other TodoMVC implementations, I don't mind adding them but I would like us to be consistent

Reply:
I simplified for now - please let me know if you'd like me to change it 😄